iteration/iteration.go

```diff
@@ -71,6 +71,17 @@ func (m Iteration) GetLastModified() time.Time {
 
 }
 
+// IsRoot Checks if given iteration is a root iteration or not
+func (m Iteration) IsRoot(spaceID uuid.UUID) bool {
```

Comment:
Please add tests for this function.

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1683/commits/b7196ecbe1a12f49ad4875cb96edc0495b9aad18

Reply:
Yeah, that looks awesome! Well done!